README.md

```diff
@@ -6,6 +6,97 @@ channel branches (e.g. `channels/8/edge`).
+
+This section provides a comparison with readily-available JRE 8 images for Ubuntu 22.04:
+ - Eclipse Adoptium: `eclipse-temurin:8u352-b08-jre-jammy`
+ - Amazon: `amazoncorretto:8u352-alpine3.14-jre`
+
+Azul Zulu does not provide a JRE image: https://hub.docker.com/r/azul/zulu-openjdk and it was not evaluated.
+Oracle does not provide official image of the Java Runtime Environment 8 and it was not evaluated.
+
+### Image size
+
+|Image|Tag|Uncompressed Size| Compressed Size|
+|-----|---|----| ----------------------------|
+| eclipse-temurin|8u352-b08-jre-jammy|221MB|80M|
+| amazoncorretto| 8u352-alpine3.14-jre| 110MB | 41M|
+| ubuntu/chiselled-jre|8-22.04_edge| 123MB|46M |
+
+
+The major points of difference with Temurin image are:
+- `/bin` and `/usr/bin` are removed, which occupy 20MB (compressed) in Temurin
+- `/var` is removed, which occupies 7.7MB due to `dpkg`
+- only minimal set of libraries is present in /usr/lib/x86_64-linux-gnu, saving 39M
+- contents of /usr/share are not present (31MB), assuming that for things like local time zone information, it is either mapped into the container, or containers run in GMT.
+
+The major points of difference with Corretto image are:
+ - Corretto deploys busybox as a shell
+ - Corretto does not have fontconfig/fonts libraries. This causes `java.awt.Font.createFont()` to fail with `java.lang.UnsatisfiedLinkError`.
+ - Corretto does not provide `glib` libraries imported by the URL proxy selector. `java.net.ProxySelector.getDefault()` call will always return `Direct Proxy`.
+
+The JRE differences itself are minimal. Th chiselled image removes libawt_xawt.so and libsplashscren.so along with accessibility support. Executables, except `java`, are removed from `jre/bin`.
+Note: chiselled docker at the moment does not provide classes.jsa (Class Data Cache) in line with Temurin JRE and it has to be generated.
+
+Below are image sizes of the deployed `acmeair` benchmark application
+|Image|Base Image|Uncompressed Size| Compressed Size|
+|-----|---|----| ----------------------------|
+| acmeair deployed to WebSphere Liberty official image| websphere-liberty:23.0.0.1-full-java8-ibmjava| 772MB| 507MB|
+| acmeair and WebSphere Liberty deployed to chiselled-jre| ubuntu/chiselled-jre:8_edge | 631MB | 407MB |
+| acmeair and WebSphere Liberty deployed to Corretto| 8u352-alpine3.14-jre | 618MB | 402MB |
+| acmeair deployed to Apache Tomcat | tomcat:9.0.71-jre8-temurin-jammy | 258MB | 108MB |
+| acmeair and Apache Tomcat deployed to chiselled-jre | ubuntu/chiselled-jre:8_edge | 164MB | 76M |
+| acmeair and Apache Tomcat deployed to Corretto| 8u352-alpine3.14-jre | 151MB | 71M |
+| acmeair Stanalone Spring Boot on Temurin | eclipse-temurin:8u352-b08-jre-jammy | 244MB | 99MB |
+| acmeair Stanalone Spring Boot | ubuntu/chiselled-jre:8_edge | 146MB | 65MB|
+| acmeair Stanalone Spring Boot on Corretto | 8u352-alpine3.14-jre | 133MB | 61MB|
+
+### Startup time
```

Comment:
For these tests, would it be possible to explain the characteristics and compute parameters leading to these values? For instance, the main questions would be:

- how many runs? is it an average over X tries?
- what was the hardware used to run these tests?
- did you try it cross architectures? if yes, did you notice any difference? (whether yes or no, this should be mentioned IMO. this would show that the solution works consistently cross architecture or if there are differences, what are they)

Reply:
Good points. I'll run tests on canonistack and update the document.